testsuites/listener/shared/client_sg/test_replication.py

```diff
@@ -227,6 +329,96 @@ def test_initial_push_replication(setup_client_syncgateway_test, continuous):
 @pytest.mark.listener
 @pytest.mark.syncgateway
 @pytest.mark.replication
+@pytest.mark.session
+@pytest.mark.parametrize("num_docs, need_attachments, replication_after_backgroundApp", [
+    (1000, True, False),
+    (10000, False, False),
+    (10000, False, False)
+])
+def test_push_replication_with_backgroundApp(setup_client_syncgateway_test, num_docs, need_attachments, replication_after_backgroundApp):
```

Comment:
col length max to 80

Reply:
I do not want to do to max 80, because it is too small and we might end of having multiple lines. I made max length as around 100